File: OfflineClient/app/src/main/java/de/jbamberger/offlineclient/ui/hackernews/HackerNewsActivity.java
package de.jbamberger.offlineclient.ui.hackernews;

import android.os.Bundle;
import android.support.design.widget.FloatingActionButton;
import android.support.design.widget.Snackbar;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

import de.jbamberger.offlineclient.R;

public class HackerNewsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_hacker_news);
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        FloatingActionButton fab = findViewById(R.id.fab);
        fab.setOnClickListener(view -> Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
                .setAction("Action", null).show());
    }

}

File: OfflineClient/app/src/main/java/de/jbamberger/offlineclient/ui/hackernews/HackerNewsActivity.kt
package de.jbamberger.offlineclient.ui.hackernews

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import de.jbamberger.offlineclient.R
import kotlinx.android.synthetic.main.activity_hacker_news.*

class HackerNewsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_hacker_news)
        setSupportActionBar(toolbar)
    }

}
